openpype/hosts/houdini/api/lib.py

```diff
@@ -559,29 +559,51 @@ def get_template_from_value(key, value):
     return parm
 
 
-def get_frame_data(node):
+def get_frame_data(node, asset_data=None, log=None):
     """Get the frame data: start frame, end frame and steps.
 
     Args:
         node(hou.Node)
 
     Returns:
-        dict: frame data for star, end and steps.
+        dict: frame data for start, end and steps.
 
     """
+    if asset_data is None:
+        asset_data = {}
+
+    if log is None:
+        log = self.log
+
     data = {}
 
     if node.parm("trange") is None:
-
+        log.debug(
+            "Node has no 'trange' parameter: {}".format(node.path())
+        )
         return data
 
     if node.evalParm("trange") == 0:
-        self.log.debug("trange is 0")
-        return data
-
-    data["frameStart"] = node.evalParm("f1")
-    data["frameEnd"] = node.evalParm("f2")
-    data["steps"] = node.evalParm("f3")
+        data["frameStartHandle"] = hou.intFrame()
+        data["frameEndHandle"] = hou.intFrame()
+        data["byFrameStep"] = 1.0
+        data["handleStart"] = 0
+        data["handleEnd"] = 0
+        log.info(
+            "Node '{}' has 'Render current frame' set. \n"
+            "Asset Handles are ignored. \n"
+            "frameStart and frameEnd are set to the "
+            "current frame".format(node.path())
+        )
```

Comment:
```suggestion
        log.info(
            "Node '{}' has 'Render current frame' set.\n"
            "Asset Handles are ignored.\n"
            "frameStart and frameEnd are set to the "
            "current frame.".format(node.path())
        )
```
Since you're adding a new line you don't need the spaces + add a dot at the end.